app/views/course/discussion/topics/_topics.html.slim

```diff
@@ -0,0 +1,4 @@
+- @topics.each do |topic|
+  = render partial: topic.actable_type.demodulize.underscore, locals: { topic: topic }
```

Comment:
OK, since the things the shared are not that much, I can remove the partial. 

The display of different types are quite similar, that's why I extracted the common part to a partial.